cmd/snap/cmd_login.go

```diff
@@ -54,28 +53,33 @@ func init() {
 }
 
 func requestLoginWith2faRetry(username, password string) error {
+	var otp []byte
+	var err error
+
+	var msgs = [3]string{
+		i18n.G("Two-factor code: "),
+		i18n.G("Bad code. Try again: "),
+		i18n.G("Wrong again. Last chance: "),
```

Comment:
s/Last chance/Once more/?

The "Last chance" makes it feel a bit like a credit card that will lock you out after three attempts.